x-pack/legacy/plugins/siem/server/lib/detection_engine/signals/build_exceptions_query.ts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { isEmpty } from 'lodash';
+
+import { Query } from '../../../../../../../../src/plugins/data/server';
+import { List, ListOperator, ListValues } from '../routes/schemas/types/lists_default_array';
+import { RuleAlertParams } from '../types';
+
+export const getLanguageBooleanOperator = (language: string, value: string) => {
+  if (language === 'lucene') {
+    return value.toUpperCase();
+  }
+
+  return value;
+};
```

Comment:
nit: This isn't a strict rule but usually within code portions where I can apply:
https://www.tomdalling.com/blog/coding-tips/coding-tip-have-a-single-exit-point/

Single entry, single exit, I do. I don't always do it, but I do try in these simple cases like so:

```ts
  if (language === 'lucene') {
    return value.toUpperCase();
  } else {
    return value;
  }
```

Just makes things easier for the next person to read imo but just optional.

Reply:
That's a good rule of thumb. Especially here where it's super easy to see the entire if else statement and understand what's going on. Updated code.